docs/app/docs/ide_integration/vscode.md

```diff
@@ -0,0 +1,65 @@
+---
+title: VSCode integration
+---
+
+
+## Java
+___
+VS Code is used for development of many different languages. In the guide below, we cover setting up a Java environment that's created inside devbox shell to work with VS Code.
+
+### Setting up Run and Debugger
+To create a devbox shell make sure to have devbox installed. If you don't have devbox installed follow the installation guide first. Then follow the steps below:
+
+1. `devbox init` if you don't have a devbox.json in the root directory of your project.
+2. `devbox add jdk` to make sure jdk gets installed in your devbox shell.
+3. `devbox shell` to activate and go into your devbox shell.
+4. `which java` and copy the path to your executable java binary inside the devbox shell. It should be something like this:
```

Comment:
We can now combine these two steps thanks to `devbox shell -- <cmd>`:

```shell
$ devbox shell -- 'command -v java'
Installing nix packages. This may take a while...
Starting a devbox shell...
/nix/store/qaf9wyzymdij19qxyw7208s72lajz65b-zulu17.34.19-ca-jdk-17.0.3/bin/java
```